includes/Story_Post_Type.php

```diff
@@ -471,6 +472,26 @@ public function filter_amp_story_element_validation_error_sanitized( $sanitized,
 		return $sanitized;
 	}
 
+	/**
+	 * Filters whether AMP-to-AMP is excluded for an element.
+	 *
+	 * The element may be either a link (`a` or `area`) or a `form`.
+	 *
+	 * @param bool       $excluded Excluded. Default value is whether element already has a `noamphtml` link relation or the URL is among `excluded_urls`.
+	 * @param string     $url      URL considered for exclusion.
+	 * @param string[]   $rel      Link relations.
+	 * @param DOMElement $element  The element considered for excluding from AMP-to-AMP linking. May be instance of `a`, `area`, or `form`.
+	 * @return bool Whether AMP-to-AMP is excluded.
+	 */
+	public function filter_amp_to_amp_linking_element_excluded( $excluded, $url, $rel, $element ) {
+		if ( $element instanceof DOMElement && $element->parentNode && 'amp-story-player' === $element->parentNode->tagName ) {
+			return true;
```

Comment:
Personal preference:

```suggestion
			$excluded = true;
```